packages/app/src/elements/login-signup.ts

```diff
@@ -489,6 +489,95 @@ export class LoginOrSignup extends StartForm {
         }
     }
 
+    protected async _migrateLegacyAccount(authResponse: {
+        email: string;
+        legacyData?: PBES2Container;
+        token: string;
+    }): Promise<boolean> {
+        const legacyData = authResponse.legacyData!;
+
+        this._submitEmailButton.start();
+
+        const choice = await alert(
+            $l(
+                "You don't have a Padloc 4 account yet but we've found " +
+                    "an account from an older version. " +
+                    "Would you like to migrate your account to Padloc 4 now?"
```

Comment:
I'd recommend switching this to a template literal instead, as it's easier to search for and less problematic than concatenations.

Reply:
The reason why I'm not using template literals within `$l()` is that they're not yet supported by our string extraction tool (https://github.com/padloc/padloc/blob/master/packages/locale/src/extract.ts#L7-L17). At some point I want to improve on that (and on our localisation tools in general) but for now it's easier to just use concatenated strings.

Reply:
I'm not saying to inject the variables on them just yet, so it'll just be a regular string to work with, no?